subsys/net/lib/openthread/platform/radio.c

```diff
@@ -43,6 +42,15 @@ LOG_MODULE_REGISTER(LOG_MODULE_NAME);
 #define FRAME_TYPE_MASK 0x07
 #define FRAME_TYPE_ACK 0x02
 
+
+enum pending_events_t {
+	pending_events_detect_energy,  /* Requested to start Energy Detection
```

Comment:
enum values are typically ALL_UPPER_CASE in Zephyr, I have seen no other. Probably enforced by the CI.